### DISCUSSION TITLE
Installation on Bare Metal

### DISCUSSION THREAD

**Discussion by Archeroftheday** [Security-Onion-Solutions/securityonion#2813] [General]
Hello, new Noob to Security Onion and have a dedicated server that I'm looking to deploy at home.  All of the install guides show spinning up Security Onion as a VM, however given it's a Linux distro - would I be able to simply create bootable media and install on my bare metal server?  Please let me know what options there are or if I'm way off base, thanks everyone!

**Comment by innovate-support** [comment]
Bare metal works better than VM in my opinion.  Stability of VM's has always been an issue for me.  But the Bare Metal systems I've used have always been stable.  More cores and more Ram is always good.

**Reply by Archeroftheday** [reply]
@innovate-support - thanks for the quick response, so do you dual boot or have a dedicated box?

**Comment by innovate-support** [comment]
No I don't dual boot things. The SO iso wants to consume everything by 
default anyways. And what would the other OS be? Please don't say Windows.

**Comment by negus6** [comment]
You can create bootable installation media from the ISO (DVD or USB) and use it to boot and install the OS and Security Onion. Windows 10 has a built in disk image burner so you can burn to DVD... so does Mac. 

Using linux you can use DD: sudo dd if=securityonion.iso of=/dev/sdx bs=1024k status=progress (where you specify the exact name of the Security Onion iso file after if= and the device corresponding to your media after of=).

Or you can use open source software, I've used Unetbootin or balenaEtcher to burn a bootable USB drives.

**Reply by launchdaemon** [reply]
Did this work for you on a recent ISO?  Do you mind me asking what hardware you are installing it on? I tried creating a USB with various tools and it would not boot but a pure Ubuntu one would.

**Reply by innovate-support** [reply]
I use a Linux Mint desktop environment, which makes this easy.  I simply have to right-click the downloaded ISO file and choose 'Make bootable USB stick'.

Windows is different though.  In that I have to use 3rd party tools like Rufus or YUMI.
https://rufus.ie
https://www.pendrivelinux.com/yumi-multiboot-usb-creator/

Also, depending on the box you're installing Security Onion on you may have to mess around with the BIOS settings to get it to boot to your USB.

**Reply by launchdaemon** [reply]
I had tried using `dd` and Etcher on Ubuntu and Rufus on Windows, also disabled Secure Boot and set the machine to use Legacy and UEFI but no luck. It would just say it couldn't find a bootable OS.  Where as an Ubuntu USB created using `dd` worked fine.
